assets/css/_dialogs.scss

```diff
@@ -170,6 +199,78 @@ $dialog-box-default-width: 790px;
   text-align: center;
 }
 
+.map-dialog {
+  width: 100vh;
+  padding: 0;
+}
+
+button.close {
+  z-index: 1002;
```

Comment:
Do you need this to be `1002`? This is a bit of a magic number. We use a z-index scale ([good reading about this here](https://medium.com/@fat/mediums-css-is-actually-pretty-fucking-good-b8e2a6c78b06)) in order to manage these numbers in a much more organized way.

Our z-index scale is defined here. https://github.com/streetmix/streetmix/blob/master/assets/css/_variables.scss#L43

We might want to set up a z-index variable at level 10 for this button and for the `.geolocation-input` element, below.